api/Metrics/Counter.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Metrics;
+
+interface Counter extends Metric
+{
```

Comment:
Should we add `@access` and `@return` doc blocks here?

Reply:
done